guides/welfare/parental_leave.md

```diff
@@ -4,4 +4,4 @@ Our current parental leave package is two weeks at full pay followed by statutor
 
 In addition to this, we offer a number of benefits, including [Remote Working](benefits/remote_working.md) and [Untracked Holiday Allowance](benefits/untracked_holiday.md). We believe these benefits provide new parents with flexibility and allow them to achieve a good balance between work and family life. 
 
-_Note: There is a strong desire to improve our parental leave offering. We're currently constrained by our company size, but as we grow and get a larger team and turnover, we're expecting to be able to make a more attractive package available._
\ No newline at end of file
+_Note: There is a strong desire to improve our parental leave offering. We're currently constrained by our company size, but as we grow and get a larger team and turnover, we're expecting to be able to make a more attractive package available._
```

Comment:
Just curious, by parental leave does this mean both paternity or maternity leave? I.e. are both of them the same (2 weeks?) and what does "followed by statutory allowance" actually mean? (Sorry, not related to your actual PR)